src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,270 @@
+      divider,
+      div({ style: { paddingTop: '0.5rem', fontSize: 16, fontWeight: 500, color: 'lightgray', ...Style.noWrapEllipsis } }, ['Result: View cache diff'])
+    ])
+  }
+
+  const compareDiffs = () => {
+    return h(Fragment, [
+      div({ style: { paddingBottom: '0.5rem', display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected workflow B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, otherWorkflowId)
+        ]),
+        h(ButtonSecondary, { style: { textTransform: 'none', paddingLeft: '1rem', height: '20px', justifyContent: 'right' }, onClick: () => resetWorkflowSelection() }, ['Reset'])
+      ]),
+      div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected call B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, otherCallFqn),
+          breadcrumbHistoryCaret,
+          div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, `index ${otherIndex}`)
+        ]),
+        h(ButtonSecondary, { style: { textTransform: 'none', paddingLeft: '1rem', height: '20px', justifyContent: 'right' }, onClick: () => resetCallSelection() }, ['Reset'])
+      ]),
+      divider,
+      div({ style: { display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, ['Result: View cache diff']),
+      diffError ?
+        h(ErrorView, { error: diffError }) :
+        (diff ?
+          [div({ style: { marginTop: '0.5rem', marginBottom: '0.5rem' } }, ['Note: the diff is expressed in terms of hashes of values rather than raw values because it is hashes that determine cache hits.']),
+            h(ReactJson, {
+              style: { whiteSpace: 'pre-wrap', border: 'ridge', padding: '0.5rem' },
+              name: false,
+              shouldCollapse: ({ name }) => { return name === 'callA' || name === 'callB' },
+              enableClipboard: false,
+              displayDataTypes: false,
+              displayObjectSize: false,
+              src: { hashDifferential: diff.hashDifferential, ...diff }
+            })] :
+          'Cache diff loading...')
+    ])
+  }
+
+  const chooseStep = () => {
+    if (!otherWorkflowId) {
+      return step1()
+    } else if (!otherCallSelected) {
+      return step2()
+    } else {
+      return compareDiffs()
+    }
+  }
+
+  return h(Modal, {
+    title: [
+      icon('search'),
+      ' Call Cache Miss Debugging Wizard'
+    ],
+    onDismiss,
+    width: '50%',
+    showButtons: false,
+    showX: true
+  }, [
+    div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
```

Comment:
This wrapper seems unnecessary. The modal content already has flex column layout and the standard amount of padding.

Reply:
Removing it seemed to make the content stick more to the sides of the modal. Feels worse to me, but maybe the layout can be recreated in a better way? (or maybe it really is better without squashing the content into the middle?)

Currently:

![image](https://user-images.githubusercontent.com/13006282/110040565-07942f00-7d11-11eb-9c43-77f51dcc7553.png)


Without the wrapper `div`:

![image](https://user-images.githubusercontent.com/13006282/110040529-f77c4f80-7d10-11eb-90d3-5839efef51a5.png)